doc/en/developer/source/eclipse-guide/index.rst

```diff
@@ -136,42 +136,33 @@ Eclipse preferences
 Code formatting
 ^^^^^^^^^^^^^^^
 
-#. Download https://raw.githubusercontent.com/geotools/geotools/master/build/eclipse/formatter.xml
-#. Navigate to ``Java``, ``Code Style``, ``Formatter`` and click ``Import...``
+The Google formatter plugin is embedded in the build and will reformat the code at each build, matching the coding conventions. Please always build before committing!
+
+The `google-java-format <https://github.com/google/google-java-format>`__ project offers a plugin for Eclipse, but we are waiting an support for ASOP variant. While we wait for this fix build once on the command line, to format the any code changes before committing.
 
-   .. image:: code_formatting1.jpg
+You may also:
 
-#. Select the ``formatter.xml`` file downloaded in step 1
+#. Download https://github.com/geotools/geotools/blob/master/build/eclipse/eclipse-java-google-style.xml
```

Comment:
this format is *close* but handles javadoc slightly differently then google formatter